sys/shell/commands/sc_vfs.c

```diff
@@ -137,6 +138,37 @@ static int _errno_string(int err, char *buf, size_t buflen)
 }
 #undef _case_snprintf_errno_name
 
+static void _print_size(uint64_t size)
+{
+    unsigned long len;
+    const char *unit;
+
+    if (size == 0) {
+        len = 0;
+        unit = NULL;
+    } else if ((size & (GiB(1) - 1)) == 0) {
+        len = size / GiB(1);
+        unit = "G";
+    }
+    else if ((size & (MiB(1) - 1)) == 0) {
+        len = size / MiB(1);
+        unit = "M";
+    }
+    else if ((size & (KiB(1) - 1)) == 0) {
+        len = size / KiB(1);
+        unit = "k";
+    } else {
+        len = size;
+        unit = NULL;
+    }
+
+    if (unit) {
```

Comment:
If you accept the suggestion, unit will never be NULL

Reply:
This was done to get the alignment right - and is still needed since the suggested unit names differ in length